pages/sdk/onramp/monerium.mdx

```diff
@@ -0,0 +1,249 @@
+})
+
+// Sign and execute the transaction as the second owner
+const transactionResult = await protocolKitOfOwner2.executeTransaction(
+  signedTx
+)
+
+// Check the transaction hash to see if the transaction settled
+console.log('transactionResult', transactionResult)
+```
+
+The `protocolKit` is an instance of the [`Safe`](../protocol-kit/reference/safe.mdx) class. 
+For more information on instantiating the `protocol-kit,` refer to the [Protocol Kit Quickstart section](../protocol-kit.mdx).
+
+### Initialize the Monerium client and authenticate the user
+
+After the message is signed and the transaction is executed, the user can authenticate with Monerium.
+
+```typescript
+import { MoneriumClient } from '@monerium/sdk'
+
+// Initialize the Monerium Client
+const monerium = new MoneriumClient({
+  clientId: 'a1b2c3-x7y8y9', // Get your client ID from Monerium
+  environment: 'sandbox' // Use the appropriate Monerium environment ('sandbox' | 'production')
+})
+
+// Start the Monerium authentication flow and send the user to Monerium
+await monerium.authorize({
+  address: safeAddress, // The address of the user's Safe
+  signature: '0x', // '0x' for Safe authentication lets Monerium look for the signature on-chain
+  redirectUrl: 'http://localhost:3000/return', // URL where Monerium will redirect the user after authenticating
+  chainId: 11155111 // Chain ID of Sepolia in this example
+})
+```
+
+Calling `authorize` will redirect the user to the Monerium login page. 
+
+### Authenticate with Monerium
+
+At Monerium, the user needs to log in or create an account. 
+Once logged in, Monerium will verify the ownership of the Safe by checking the signed message. 
+After successful verification, Monerium will create an IBAN and link it to the Safe. 
+The user will then be redirected to the specified `redirectUrl` with an added `code` query parameter.
+
+### Finish the authentication
+
+Once the user lands back on your page, finish the authentication process.
+
+```typescript
+// Returns true, if the user is authorized
+const isAuthorized = await monerium.getAccess()
+```
+
+Congratulations, you authenticated your user with Monerium and linked the Safe to the Monerium account.
+</Steps>
+
+## Step 2: Place an Order
+
+Now that the user is authenticated with Monerium, you can place an order to transfer tokens from the Safe to an IBAN.
+
+<Steps>
+
+### Get some tokens
+
+First, your user needs to obtain some EURe test tokens on Sepolia:
+
+* Log into the [Monerium Sandbox](https://sandbox.monerium.dev/accounts) account.
+* Click on the `ADD MONEY` button.
+* Create a test IBAN transfer onto the account. The tokens from this test transfer will be available in the Safe.
+
+<Image src={MoneriumAddMoney} width={300} height={500}  />
+
+### Send an order to Monerium
+
+To send tokens from the Safe to an IBAN, the user must sign a message with the Safe and send the order to Monerium. 
+You can trigger both actions simultaneously.
```

Comment:
I would rephrase it this way for simplicity and coherence with the order of our steps:
```suggestion
To send tokens from a Safe to an IBAN, users must send the order to Monerium and sign a message with their account.
```